docs/skopeo-copy.1.md

```diff
@@ -93,6 +93,11 @@ Do not copy signatures, if any, from _source-image_. Necessary when copying a si
 
 Add a “simple signing” signature using that key ID for an image name corresponding to _destination-image_
 
+**--sign-by-sigstore** _param-file_
+
+Add a sigstore signature based on further optinos specified in a containers sigstore signing parameter file _param-file_.
```

Comment:
```suggestion
Add a sigstore signature based on further options specified in a container's sigstore signing parameter file _param-file_.
```

Reply:
Thanks, fixed the `options` typo.

WRT the other part, this is not “a parameter file owned by / related to a container”, it’s “a signing parameter file defined by the `GitHub.com/containers` organization (i.e. not a `cosign`-defined parameter file).

I fully appreciate that a “containers sigstore signing parameter file” is a horrible mouthful that desperately needs replacing; what would be a better name? Compare a tiny bit more discussion in https://github.com/containers/image/pull/1787 . Ultimately the name for this concept is decided by the man page in that other PR, and this one would follow the lead.

Reply:
Oh, that's entertaining.  Thanks for the explanation.  I'll try another whack at it.

Reply:
“entertaining”. Seriously, I would love to replace that name.